constructor/nsis/main.nsi.tmpl

```diff
@@ -803,7 +811,12 @@ Section "Install"
     @PKG_COMMANDS@
 
     # Cleanup
-    Delete "$INSTDIR\pkgs\env.txt"
+    SetOutPath "$INSTDIR"
+    ${If} $ARGV_KeepPkgCache == "1"
+        Delete "$INSTDIR\pkgs\env.txt"
+    ${Else}
+        RMDir /r "$INSTDIR\pkgs"
```

Comment:
This is where the package cache is now removed. Note the use of the `/r` flag; without it, it will not work. Furthermore, the `SetOutPath` command a couple of lines up is necessary to remove the directory itself, because without it nsis will be inside `$INSTDIR\pkgs`. Remove that command, and the _contents_ of `$INSTDIR\pkgs` will be removed, but not the directory itself (which frankly would be fine).